src/components/DataDrawer/Chart/index.tsx

```diff
@@ -116,6 +121,7 @@ function formatChartData(data: TableData, config: ChartConfig) {
         backgroundColor: colors[i],
         borderColor: colors[i],
         borderWidth: 2,
+        pointRadius: data.EWSConfig ? 0 : undefined,
```

Comment:
Add a small comment explaining what's happening?

Reply:
if you keep it undefined doesnt it get overridden in datasetsWithTresholds already?